OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -257,23 +257,71 @@ bool TracerThread::OpenSampling(const std::vector<int32_t>& cpus) {
   return true;
 }
 
-bool TracerThread::InitGpuTracepointEventProcessor() {
-  int amdgpu_cs_ioctl_id = GetTracepointId("amdgpu", "amdgpu_cs_ioctl");
-  if (amdgpu_cs_ioctl_id == -1) {
-    return false;
+bool TracerThread::OpenRingBuffersForTracepointAndRedirectOnExistingIfNecessary(
```

Comment:
Can this easily be split into two methods? The name already indicates this method is doing too many things. 

OpenRingBufffersForTracepoint()
RedirectOnExistingBufferIfPossible()

I would also suggest to change Necessary to Possible. It's not necessary to redirect onto an existing ring buffer but it's advantageous to do so.